File: 프로그래머스/unrated/120846. 합성수 찾기/합성수 찾기.py
def solution(n):
    answer = 0
    for i in range(1, n+1):
        flag = 0
        for j in range(1, i+1):
            if i % j == 0:
                flag += 1
                if flag == 3:
                    answer += 1
                    break
    return answer

File: 프로그래머스/lv0/120846. 합성수 찾기/합성수 찾기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        int cnt = 0;
        String str = "";
        for (int i = 3; i <= n; i++) {
            for (int j = 1; j <= i; j++) {
                if (i % j == 0) {
                    cnt++;
                }
                if (cnt == 3) {
                    cnt = 0;
                    answer++;
                    break;
                }
            }
        }
        return answer;
    }
}